README.markdown

```diff
@@ -24,7 +24,9 @@ gitlab-le \
 --email      example@example.com     `# Let's Encrypt email address` \
 --domain     example.com             `# Domain that the cert will be issued for` \
 --repository gitlab_user/gitlab_repo `# Namespaced repository identifier` \
---token      ...                     `# GitLab personal access token, see https://gitlab.com/profile/personal_access_tokens`
+--token      ...                     `# GitLab personal access token, see https://gitlab.com/profile/personal_access_tokens` \
+--jekyll     true                    `# Set true if use Jekyll` \
```

Comment:
It shouldn't be necessary to specify `true` if the option was defined as a boolean. Simply having the argument should be enough to toggle the flag.